kolibri/plugins/coach/assets/src/views/learners/LearnerSummaryPage/index.vue

```diff
@@ -190,7 +186,10 @@
         return this.getLearnersForExam(quiz).includes(this.learner.id);
       },
       quizLink(quizId) {
-        return this.classRoute(PageNames.REPORTS_LEARNER_REPORT_QUIZ_PAGE_ROOT, { quizId });
+        return this.classRoute(PageNames.QUIZ_LEARNER_REPORT, { quizId });
```

Comment:
Here this route is failing because we are not passing the questionId, interactionIndex and tryIndex params. Here it would be better to use the [QUIZ_LEARNER_PAGE_ROOT page](https://github.com/learningequality/kolibri/blob/a85c02557be5f12f7c23422964d8d1e3a8ea835c/kolibri/plugins/coach/assets/src/routes/examRoutes.js#L94) that the only thing that it does is that it redirects it to the `QUIZ_LEARNER_REPORT` page but defines default values for these params, so we dont need to do it here.